microsoft-edge/webview2/release-notes.md

```diff
@@ -36,6 +36,121 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+* [CoreWebView2.RemoveWebResourceRequestedFilterWithRequestSourceKinds Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.removewebresourcerequestedfilter?view=webview2-dotnet-1.0.1293.44#microsoft-web-webview2-core-corewebview2-removewebresourcerequestedfilter)
+* [CoreWebView2.AddWebResourceRequestedFilter Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter?view=webview2-dotnet-1.0.1340-prerelease)
+* [CoreWebView2WebResourceRequestedEventArgs.RequestedSourceKind Property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webresourcerequestedeventargs.requestedsourcekind?view=webview2-dotnet-1.0.1340-prerelease)
+* [CoreWebView2WebResourceRequestSourceKinds Enum](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webresourcerequestsourcekinds?view=webview2-dotnet-1.0.1340-prerelease)
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.AddWebResourceRequestedFilterWithRequestSourceKinds Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1340-prerelease#addwebresourcerequestedfilter)
+* [CoreWebView2.RemoveWebResourceRequestedFilter Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1340-prerelease#removewebresourcerequestedfilter)
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [AddWebResourceRequestedFilterWithRequestSourceKinds method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimental16?view=webview2-1.0.1340-prerelease&preserve-view=true#addwebresourcerequestedfilterwithrequestsourcekinds)
+* [RemoveWebResourceRequestedFilterWithRequestSourceKinds method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimental16?view=webview2-1.0.1340-prerelease&preserve-view=true#removewebresourcerequestedfilterwithrequestsourcekinds)
+* [ICoreWebView2ExperimentalWebResourceRequestedEventArgs.RequestedSourceKind property](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebresourcerequestedeventargs?view=webview2-1.0.1340-prerelease&preserve-view=true#get_requestedsourcekind:~:text=Descriptions-,get_RequestedSourceKind,-The%20web%20resource)
+
+---
+
+*  Added support for custom scheme registration which allows WebView2 apps to be able to handle `WebResourceRequested` event for requests with the specified scheme and be able to navigate the WebView2 control to the custom scheme. 
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+
+
+##### [Win32/C++](#tab/win32cpp)
+

```

Comment:
Should we delete this tab-set?  because both list items are for the same API: WebResourceRequested event & possibly a couple related API links.  Check the PM's commits (PR UI: the "Changes from commits" menu in upper left) to see which API items were specified in the C++ tab in the tab-sets for the two list items.

Reply:
Here are the C++ links that were originally specified for the two `WebResourceRequested` list items:

*  Added support for `webResourceRequested` for workers which allows setting filters in order to receive `WebResourceRequested` events for service workers, shared workers, and different origin iFrames.
* [ICoreWebView2experimental16](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimental16?view=webview2-1.0.1340-prerelease)
* [ICoreWebView2ExperimentalWebResourceRequestedEventArgs](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebresourcerequestedeventargs?view=webview2-1.0.1340-prerelease)

*  Added support for custom scheme registration which allows WebView2 apps to be able to handle `WebResourceRequested` event for requests with the specified scheme and be able to navigate the WebView2 control to the custom scheme. 
* [ICoreWebView2ExperimentalEnvironmentOptions](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions?view=webview2-1.0.1340-prerelease)
* [ICoreWebView2ExperimentalCustomSchemeRegistration](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcustomschemeregistration?view=webview2-1.0.1340-prerelease)